src/aktualizr_secondary/aktualizr_secondary_interface.h

```diff
@@ -13,7 +13,10 @@ class IAktualizrSecondary {
   virtual std::tuple<Uptane::EcuSerial, Uptane::HardwareIdentifier, PublicKey> getInfo() const = 0;
   virtual Uptane::Manifest getManifest() const = 0;
   virtual bool putMetadata(const Uptane::RawMetaPack& meta_pack) = 0;
+
   virtual bool sendFirmware(const std::string& firmware) = 0;
+
+  virtual data::ResultCode::Numeric sendFirmware(const uint8_t* data, size_t size) = 0;
```

Comment:
This class (`IAktualizrSecondary`) appears unused now. Can we get rid of it?

Reply:
yes, I thought, I removed it, it must be result of incorrect rebasing with master.